stubs/Flask-Migrate/flask_migrate/__init__.pyi

```diff
@@ -0,0 +1,105 @@
+from collections.abc import Callable
+from logging import Logger
+from typing import Any, TypeVar
+from typing_extensions import ParamSpec, TypeAlias
+
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import MetaData
+
+_T = TypeVar("_T")
+_P = ParamSpec("_P")
+_App: TypeAlias = Any  # flask.Flask is not possible as a dependency yet
+
+alembic_version: tuple[int, int, int]
+log: Logger
+
+class _MigrateConfig:
+    migrate: Migrate
+    db: SQLAlchemy | None
+    directory: str
+    configure_args: dict[str, Any]
+    def __init__(self, migrate: Migrate, db: SQLAlchemy | None, **kwargs) -> None: ...
+    @property
+    def metadata(self) -> MetaData: ...
+
```

Comment:
We usually try to avoid including private things in the stubs, unless there's a specific reason to make an exception. Is there a specific reason why this needs to be included?

```suggestion
```